.golangci.yml

```
@@ -84,6 +84,9 @@ linters:
       - linters:
           - revive
         text: "should have comment or be unexported"
+      - linters:
+          - revive
+        text: "avoid meaningless package names"
```

```suggestion
        text: "var-naming: avoid meaningless package names"
```
To put some more context into this exclusion. But it would be even better if the revive linter is reconfigured instead, ref. https://golangci-lint.run/usage/linters/#revive

---

Thank you 🥇🚀
I accept your suggestion, but I’m not sure how to reconfigure it myself. (I would need spend some time checking it out) Maybe we can add an issue in the repo, so I or someone else can check it out later and try to improve how it works.